### PR TITLE
feature/standing_detail (Create Banner)

### PR DESCRIPTION
**Pull request by nhanviet1**
## Related Tickets
- [#Task 57035: Content](https://edu-redmine.sun-asterisk.vn/issues/57035)
## WHAT
- Create the banner for Driver's detail screen
- Get data 
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DetailFragment.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.example.formula1.view.mainscreen.fragments.standing.driverdetail
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.formula1.R
+
+class DetailFragment : Fragment() {
```
Comment (minhnn-1279) [inline app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DetailFragment.kt]: Viết base fragment nhé

File: app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingDetail.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            fragmentList
+        )
+    }
+    private val viewModel: StandingViewModel by viewModel()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setup()
+        observeData()
+        getData()
```
Comment (minhnn-1279) [inline app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingDetail.kt:35]: mấy cái method này để abstract ở base nhé

File: app/src/main/res/layout/activity_driver_standing_detail.xml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            android:layout_height="@dimen/dp_100"
+            android:layout_marginStart="@dimen/dp_10"
+            android:layout_marginBottom="@dimen/dp_20"
+            android:background="@color/color_white"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent" />
+
+        <TextView
+            android:id="@+id/text_name"
+            android:layout_width="0dp"
```
Comment (hungld-1302) [inline app/src/main/res/layout/activity_driver_standing_detail.xml]: dimen

File: app/src/main/res/layout/fragment_detail.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    android:layout_height="match_parent"
+    tools:context=".view.mainscreen.fragments.standing.driverdetail.DetailFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="Detail Fragment" />
+
+</FrameLayout>
```
Comment (minhnn-1279) [inline app/src/main/res/layout/fragment_detail.xml]: endline

File: app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingFragment.kt [hunk 5]
```diff
@@ -90,7 +94,13 @@ class DriverStandingFragment : Fragment() {
     }
 
     private fun onItemClick(item: Response) {
-        //Do later
-        item
+        val intent = Intent(requireActivity(), DriverStandingDetail::class.java)
```
Comment (minhnn-1279) [inline app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingFragment.kt]: check null context mới cho chuyển hay ko nhé, require dễ gây crash app

File: app/src/main/res/layout/activity_driver_standing_detail.xml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+        android:background="@color/color_black"
+        app:tabTextColor="@color/color_white"
+        app:tabTextAppearance="@style/custom_tab_text"
+        app:tabIndicatorColor="@color/color_red"/>
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/viewPager"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
```
Comment (hungld-1302) [inline app/src/main/res/layout/activity_driver_standing_detail.xml:101]: .

File: app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingDetail.kt (hunk 2 above)
Comment (nhanviet1) [reply app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingDetail.kt:35]: Có mấy màn activity của tôi không có mấy method này đâu 🐧

**Pull request merged**